### PR TITLE
fixing #5283

### PR DESCRIPTION
**Pull request by averrin**
change toolbar-nav.help-button color
header-help-button2

### CODE REVIEW

**Comment by averrin** [comment]
so, i again write wrong issue number. :cry:

File: website/views/shared/header/menu.jade [hunk 1]
```diff
@@ -146,8 +146,9 @@ nav.toolbar(ng-controller='AuthCtrl', ng-class='{active: isToolbarHidden}')
               a(target="_blank" ng-href='http://data.habitrpg.com?uuid={{user._id}}')=env.t('dataTool')
             li
               a(ui-sref='options.settings.export')=env.t('exportData')
-      li.toolbar-button-dropdown
+      li.toolbar-button-dropdown.help-button
         a(target="_blank" href='http://habitrpg.wikia.com/wiki/')
+          span.glyphicon.glyphicon-question-sign
```
Comment (crookedneighbor) [inline website/views/shared/header/menu.jade:151]: Instead of adding a margin, just make this: `span.glyphicon.glyphicon-question-sign &nbsp;`

**Comment by crookedneighbor** [comment]
I do like the inclusion of the ? mark glyphicon, but I would still prefer this to be a standbard bootstrap button class: http://getbootstrap.com/css/#buttons-options

@lemoness Do you have a preference for which color it should be?

**Comment by crookedneighbor** [comment]
Uhg... I forgot that we customized the toolbar to not use normal bootstrap buttons. This is going to be harder than I thought.

**Comment by averrin** [comment]
More harder. And standart bootstrap color is too bright. 
So i Can add nbsp,  but imho using html entities for styling - bad idea.

**Comment by averrin** [comment]
header-help-button2
#d0eed0
header-help-button-c0e0ff
 #c0e0ff
header-help-button-ffe0c0
#ffe0c0

**Comment by crookedneighbor** [comment]
Decided to go with a different solution, but you had a good idea adding the ? mark to the button and I'm considering the pr that did get in a collaboration with you, so this will count towards a contributor tier. 

What's your User ID? You can find it in [Settings > API](https://habitrpg.com/#/options/settings/api). Be sure to only copy the User ID and _not_ your API Token.

**Pull request closed without merging**